app/src/main/java/com/example/tkey_android/FirstFragment.java

```diff
@@ -163,557 +222,642 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
             }
         });
 
-        binding.requestNewShare.setOnClickListener(view1 -> {
+//        binding.requestNewShare.setOnClickListener(view1 -> {
+//            try {
+//                activity.transferStorage = new StorageLayer(false, "https://metadata.tor.us", 2);
+//                activity.transferProvider =   new ServiceProvider(true, activity.postboxKey,true, verifier, verifierId, nodeDetail);
+//                activity.transferKey = new ThresholdKey(null, null, activity.transferStorage, activity.transferProvider, null, null, false, false);
+//                activity.transferKey.initialize(null, null, false, false, result -> {
+//                    if (result instanceof Result.Error) {
+//                        Exception e = ((Result.Error<KeyDetails>) result).exception;
+//                        renderError(e);
+//                    } else if (result instanceof Result.Success) {
+//                        requireActivity().runOnUiThread(() -> {
+//                            String userAgent = new WebView(getContext()).getSettings().getUserAgentString();
+//                            SharetransferModule.requestNewShare(activity.transferKey, userAgent, "[]", (result1) -> {
+//                                if (result1 instanceof Result.Error) {
+//                                    renderError(((Result.Error<String>) result1).exception);
+//                                } else if (result1 instanceof Result.Success) {
+//                                    String requestId = ((Result.Success<String>) result1).data;
+//                                    REQUEST_ID = requestId;
+//                                    requireActivity().runOnUiThread(() -> {
+//                                        Snackbar snackbar = Snackbar.make(view1, "Request Id: " + requestId, Snackbar.LENGTH_LONG);
+//                                        snackbar.show();
+//                                        binding.requestNewShare.setEnabled(false);
+//                                        binding.lookForRequests.setEnabled(true);
+//                                        binding.cleanupRequests.setEnabled(true);
+//                                    });
+//                                }
+//                            });
+//                        });
+//                    }
+//                });
+//            } catch (RuntimeError | JSONException e) {
+//                renderError(e);
+//            }
+//        });
```

Comment:
Should probably restore all this functionality and move the tss signing to a new fragment so that it matches the swift example. The share transfer functionality is validly removed in this PR though. This comment applies to all commented code in this PR.

Reply:
will be taken care by devrel